voluncheer/profiles/test_models.py

```diff
@@ -1,16 +1,12 @@
-from django.test import Client
 from django.test import TestCase
-from django.urls import reverse
 
 from profiles.models import Organization
 from profiles.models import User
 from profiles.models import UserType
 from profiles.models import Volunteer
 
-from map import urls
-from map import views
 
-'''
+"""
 Commented out for now until I figure out the namespace and app_name diff
```

Comment:
If you are already testing for the view below, then these tests for URLs might not be necessary. Pls double-check and remove if that's the case.

Reply:
Yes, thank you for pointing out. Removed.